nireports/tests/conftest.py

```diff
@@ -57,3 +58,10 @@ def nthreads():
 
     # Tests are linear, so don't worry about leaving space for a control thread
     return min(int(getenv("CIRCLE_NPROCS", "8")), cpu_count())
+
+
+@pytest.fixture()
```

Comment:
Could save ourselves the trouble of remembering to add it to new tests just by making it auto-use:

```suggestion
@pytest.fixture(autouse=True)
```

I assume `plt.close("all")` doesn't warn or error if there are no plots...